HackerRank/StringManipulationTests/AlternatingCharactersTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using StringManipulation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace StringManipulation.Tests
+{
+    [TestClass()]
+    public class AlternatingCharactersTests
+    {
+        [TestMethod()]
+        public void alternatingCharTest1()
```

Comment:
Make method names capital. Also indicate if what type of test it is in the name.